api/RTCErrorEvent.json

```diff
@@ -29,10 +29,10 @@
             "version_added": null
           },
           "safari": {
-            "version_added": null
+            "version_added": false
           },
           "safari_ios": {
-            "version_added": "6"
+            "version_added": false
```

Comment:
Yep, no RTCErrorEvent in WebKit source. The "6" there was definitely wrong, Safari iOS back in 2012 did not have anything around WebRTC :)